### PR TITLE
Add textarea field type

### PR DESCRIPTION
**Pull request by jfurrow**
This adds the ability to render a `textarea` with the same behavior as other `input` elements.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @kennyt

File: src/Form/FieldTypes.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 import FieldInput from './FieldInput';
 import FieldPassword from './FieldPassword';
+import FieldTextarea from './FieldTextarea';
 
 const FieldTypes = {
   'checkbox': FieldCheckbox,
+  'password': FieldPassword,
   'text': FieldInput,
-  'password': FieldPassword
+  'textarea': FieldTextarea
```
Comment (jfurrow) [inline src/Form/FieldTypes.js]: :abcd:

File: src/Form/FieldTextarea.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+export default class FieldTextarea extends FieldInput {
+  getInputElement(attributes) {
+    let {props} = this;
+
+    let classes = classNames(props.inputClass, props.sharedClass);
+    attributes = this.bindEvents(attributes);
+
+    if (this.isEditing() || props.writeType === 'input') {
+      return (
```
Comment (kennyt) [inline src/Form/FieldTextarea.js]: did we want to also wrap this inside of a `.render` call or no?

Comment (mlunoe) [inline src/Form/FieldTextarea.js]: Also aren't we repeating FieldInput here?
Could we make it a bit more flexible by adding the option of rendering different tags (when editing) for a core text field component and then extending that to create a FieldInput and FieldTextarea.
Just an idea

Comment (jfurrow) [inline src/Form/FieldTextarea.js]: @kennyt Yeah, I'll add the render wrapper here too.

@mlunoe You are right, but I am concerned about doing this in the time frame we have... Would you be okay with me creating a JIRA task for this, which can be completed when the installer is further along? Others, feel free to weigh in here.

**Comment by kennyt** [comment]
alright this looks good to me

File: src/Form/FieldTextarea.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
+
+import FieldInput from './FieldInput';
+import IconEdit from './icons/IconEdit';
+
+export default class FieldTextarea extends FieldInput {
```
Comment (rcorral) [inline src/Form/FieldTextarea.js]: `module.exports =`

**Comment by rcorral** [comment]
looks good after that comment

**Comment by jfurrow** [comment]
@rcorral Done!

**Pull request merged**